js/main.js

```diff
@@ -16,3 +18,77 @@ function processData(data, title, id) {
   lineplot = document.getElementById(id);
 	Plotly.plot(lineplot, [data], {title: title, showlegend: true});
 }
+
+function loadCSVGraph() {
```

Comment:
IMO function name should be more specific, maybe "loadCSVPredictions" or so, as this is not a general CSV loader function